pkgs/os-specific/linux/nsncd/default.nix

```diff
@@ -39,5 +39,6 @@ rustPlatform.buildRustPackage {
     ];
     # never built on aarch64-darwin, x86_64-darwin since first introduction in nixpkgs
     broken = stdenv.hostPlatform.isDarwin;
+    updateScript = nix-update-script { };
```

Comment:
Should be under passthru, not under meta.

Reply:
:facepalm: good catch!